source/adapter.screenshare.js

```diff
@@ -0,0 +1,161 @@
+(function () {
+
+  'use strict';
+
+  var tempGetUserMedia = null;
+
+  if (window.navigator.mozGetUserMedia) {
```

Comment:
I'm uncomfortable with the fact that we check 
window.navigator.mozGetUserMedia here
and navigator.mozGetUserMedia in the basic adapter (I know it's the same)

I feel like we should either factorise the code of store the first check somewhere (which we do actually, in webrtcDetectedBrowser)

@serrynaimo Question: do we want adapter.screenshare.js to be a standalone or does it need the adapter.js base code?